news/migrations/0041_alter_newsarticletag_tag.py

```diff
@@ -7,7 +7,6 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('taggit', '0004_alter_taggeditem_content_type_alter_taggeditem_tag'),
```

Comment:
did you manually change this or was it part of `makemigrations`?

Reply:
It was part of the migrations